gradle/dependencies.gradle

```diff
@@ -14,7 +14,7 @@ ext {
     spockVersion = '1.2-groovy-2.4'
     groovyVersion = '2.4.15'
     byteBuddyVersion = '1.8.21'
-    altPodsVersion = '1.16.0-SNAPSHOT'
+    altPodsVersion = '1.17.0-SNAPSHOT'
```

Comment:
I am using 1.26.0-SNAPSHOT in my app. where all google signing fails.

Reply:
1.26.0-SNAPSHOT is "to new". Google sign-in api had some breaking changes, and sign-in flow works a bit different. There is no way to simple bump version to 1.26. More changes in the code are required